e2e-tests/auth/signin.js

```diff
@@ -0,0 +1,131 @@
+var environment = require('../globel_env.js');
+
+describe('Sign in',() => {
+  const emailInput = element(by.css('[ng-model="signin.credentials.email"]'));
+  const passwdInput = element(by.css('[ng-model="signin.credentials.password"]'));
+  const signinButton = element(by.css('[ng-click="signin.submit()"]'));
+  const env = new environment();
+
+  browser.getProcessedConfig().then((config) => {
+    env.setUser(config.capabilities.browserName+"-"+config.capabilities.os);
+  });
+
+  beforeEach(() => {
+    browser.get(env.getWeb());
+  });
+
+  describe('When users entry this website : ',() => {
+    it('Should check website title is \"Log in to your account\".',() => {
+      browser.getCurrentUrl().then((result) => {
+        expect(result).toBe('http://10.26.1.63:3000/auth/signin');
+      });
+    });
+  });
+
+  describe('When users not input email and password : ',() => {
+    it('Should check [SIGN IN] will disabled.',() => {
+      expect(signinButton.isEnabled()).toBe(false);
+    });
+  });
+
+  describe('When users input null in email and password : ',() => {
+  	beforeEach(() => {
+      emailInput.sendKeys();
+      passwdInput.sendKeys();
+      emailInput.sendKeys();
+  	});
+    it('Should check [SIGN IN] will disabled.',() => {
+      expect(signinButton.isEnabled()).toBe(false);
+    });
+    it('Should check message will show that \"You left the field blank\".',() => {
+      element(by.css('[ng-if="signin.form.email.$touched"]')).getAttribute('class').then((result) => {
+        expect(result).toContain('ng-active');
+      });
+      element(by.css('[ng-if="signin.form.email.$touched"]')).getText().then((result) => {
+        expect(result).toBe('You left the field blank.');
```

Comment:
Check whether the message element is shown or hidden would be better.
The expect of the whole message in English will failed when the interface is set in Chinese or other language.